transports/noise/src/protocol.rs

```diff
@@ -90,7 +90,7 @@ impl Keypair {
         self,
         id_keys: &identity::Keypair,
     ) -> Result<AuthenticKeypair, Error> {
-        let sig = id_keys.sign(&[STATIC_KEY_DOMAIN.as_bytes(), self.public.as_ref()].concat())?;
+        let sig = id_keys.sign(&[STATIC_KEY_DOMAIN.as_bytes(), self.public.as_ref()].concat()).unwrap();
```

Comment:
ditto; we should use `expect` here to provide context if it ever does fails and panics. Unlikely though but its more of a just-in-case.